crypto/src/main/scala/coop/rchain/crypto/util/Sorting.scala

```diff
@@ -0,0 +1,13 @@
+package coop.rchain.crypto.util
+
+import coop.rchain.crypto.PublicKey
+import coop.rchain.crypto.codec.Base16
+
+object Sorting {
+
+  implicit val byteArrayOrdering: Ordering[Array[Byte]] =
```

Comment:
can we make this private, and maybe not even implicit?

Reply:
Just because it isn't used anywhere?

Reply:
👍